library/core/src/unicode/unicode_data.rs

```diff
@@ -580,1815 +580,804 @@ pub mod white_space {
 
 #[rustfmt::skip]
 pub mod conversions {
+    const INDEX_MASK: u32 = 4194304;
```

Comment:
nit: might be clearer if this were `0x40_0000` instead, at least for me who only knows my powers of two up so high.

Reply:
I agree, pushed a fix to `0x400000` at least (dunno if there's some fmt operator that inserts `_`s?).

Reply:
Ultimately this is the generated code, so I'm not too worried about readability in it.